### PR TITLE
Spark and Trino setup for attaching a debugger

### PR DESCRIPTION
**Pull request by xBis7**
For the spark debugger setup to go into effect, you have to rebuild the spark image. The easiest way is to remove the old image and then let docker-compose build it again during the first environment startup.

```
docker image ls | grep spark
```
get the image id

```
docker image rm <id>
```

### CODE REVIEW

**Comment by xBis7** [comment]
For the spark debugger setup to work, you have to rebuild the spark image. The easiest way is to remove the old image and then let docker-compose build it again during the first environment startup.

```
docker image ls | grep spark
```
get the image id

```
docker image rm <id>
```

**Comment by GeorgeJahad** [comment]
in general I prefer instructions like the above to be in a readme in the pr, instead of a comment on the pr; (future users will see the readme but won't see the pr.)

**Comment by mladjan-gadzic** [comment]
> in general I prefer instructions like the above to be in a readme in the pr, instead of a comment on the pr; (future users will see the readme but won't see the pr.)

Could you please clarify what did you mean by this? Is it PR description or README file in a repo?

File: compose/spark/conf/spark-class [hunk 1]
```diff
@@ -0,0 +1,122 @@
+#!/usr/bin/env bash
```
Comment (mladjan-gadzic) [inline compose/spark/conf/spark-class:1]: Why do we need this?

Comment (xBis7) [reply compose/spark/conf/spark-class:1]: I added a comment with an explaination. Check line 20.

**Comment by GeorgeJahad** [comment]
> Could you please clarify what did you mean by this? Is it PR description or README file in a repo?

User documention should be readme's, not comments in the PR description.

**Review by GeorgeJahad — DISMISSED** [review]

**Comment by GeorgeJahad** [comment]
where are the readme changes?

**Comment by GeorgeJahad** [comment]
go ahead and merge after adding the readme

**Pull request merged**